<Type Name="AsynchronousOperationException" FullName="System.IdentityModel.AsynchronousOperationException">
  <Metadata>
    <Meta Name="ms.openlocfilehash" Value="0557086981880cebb05d286f3f156744916cb525" />
    <Meta Name="ms.sourcegitcommit" Value="9e06b87bcfa4347dbbaaa4b3cc36b43b93b5badb" />
    <Meta Name="ms.translationtype" Value="HT" />
    <Meta Name="ms.contentlocale" Value="pt-BR" />
    <Meta Name="ms.lasthandoff" Value="11/20/2018" />
    <Meta Name="ms.locfileid" Value="52245819" />
  </Metadata>
  <TypeSignature Language="C#" Value="public class AsynchronousOperationException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi serializable beforefieldinit AsynchronousOperationException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:System.IdentityModel.AsynchronousOperationException" />
  <TypeSignature Language="VB.NET" Value="Public Class AsynchronousOperationException&#xA;Inherits Exception" />
  <TypeSignature Language="C++ CLI" Value="public ref class AsynchronousOperationException : Exception" />
  <TypeSignature Language="F#" Value="type AsynchronousOperationException = class&#xA;    inherit Exception" />
  <AssemblyInfo>
    <AssemblyName>System.IdentityModel</AssemblyName>
    <AssemblyVersion>4.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute FrameworkAlternate="netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8">
      <AttributeName>Serializable</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>
      <span data-ttu-id="5d188-101">A exceção gerada quando ocorre um erro durante uma operação assíncrona.</span>
      <span class="sxs-lookup">
        <span data-stu-id="5d188-101">The exception that is thrown when an error occurs during an asynchronous operation.</span>
      </span>
    </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <MemberGroup MemberName=".ctor">
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Docs>
        <summary>
          <span data-ttu-id="5d188-102">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-102">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class.</span>
          </span>
        </summary>
      </Docs>
    </MemberGroup>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException ();" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.AsynchronousOperationException.#ctor" />
      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException();" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters />
      <Docs>
        <summary>
          <span data-ttu-id="5d188-103">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" />.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-103">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="5d188-104">Este construtor inicializa o <xref:System.Exception.Message> propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro.</span><span class="sxs-lookup"><span data-stu-id="5d188-104">This constructor initializes the <xref:System.Exception.Message> property of the new instance to a system-supplied message that describes the error.</span></span> <span data-ttu-id="5d188-105">Esta mensagem leva em conta a cultura do sistema atual.</span><span class="sxs-lookup"><span data-stu-id="5d188-105">This message takes into account the current system culture.</span></span>  
  
 <span data-ttu-id="5d188-106">A tabela a seguir mostra os valores de propriedade inicial para a nova instância.</span><span class="sxs-lookup"><span data-stu-id="5d188-106">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="5d188-107">Propriedade</span><span class="sxs-lookup"><span data-stu-id="5d188-107">Property</span></span>|<span data-ttu-id="5d188-108">Valor</span><span class="sxs-lookup"><span data-stu-id="5d188-108">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="5d188-109">`null`.</span><span class="sxs-lookup"><span data-stu-id="5d188-109">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="5d188-110">A cadeia de caracteres da mensagem de erro localizada.</span><span class="sxs-lookup"><span data-stu-id="5d188-110">The localized error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.AsynchronousOperationException.#ctor(System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.AsynchronousOperationException : Exception -&gt; System.IdentityModel.AsynchronousOperationException" Usage="new System.IdentityModel.AsynchronousOperationException innerException" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="innerException">
          <span data-ttu-id="5d188-111">A <see cref="T:System.Exception" /> que é a causa da exceção atual.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-111">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="5d188-112">Caso o parâmetro <paramref name="innerException" /> não seja <see langword="null" />, a exceção atual é acionada em um bloco <see langword="catch" /> que identifica a exceção interna.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-112">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="5d188-113">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" /> com uma mensagem de erro fornecida pelo sistema e uma referência à exceção interna que é a causa dessa exceção.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-113">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class with a system-supplied error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="5d188-114">Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência para a exceção anterior na propriedade <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="5d188-114">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="5d188-115">O <xref:System.Exception.InnerException%2A> propriedade retorna o mesmo valor que é passado para o construtor.</span><span class="sxs-lookup"><span data-stu-id="5d188-115">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="5d188-116">A tabela a seguir mostra os valores de propriedade inicial para a nova instância.</span><span class="sxs-lookup"><span data-stu-id="5d188-116">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="5d188-117">Propriedade</span><span class="sxs-lookup"><span data-stu-id="5d188-117">Property</span></span>|<span data-ttu-id="5d188-118">Valor</span><span class="sxs-lookup"><span data-stu-id="5d188-118">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="5d188-119">A referência de exceção interna.</span><span class="sxs-lookup"><span data-stu-id="5d188-119">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="5d188-120">Uma mensagem fornecida pelo sistema que descreve o erro.</span><span class="sxs-lookup"><span data-stu-id="5d188-120">A system-supplied message that describes the error.</span></span> <span data-ttu-id="5d188-121">Esta mensagem leva em conta a cultura do sistema atual.</span><span class="sxs-lookup"><span data-stu-id="5d188-121">This message takes into account the current system culture.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.AsynchronousOperationException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.AsynchronousOperationException : string -&gt; System.IdentityModel.AsynchronousOperationException" Usage="new System.IdentityModel.AsynchronousOperationException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="5d188-122">A mensagem de erro que explica a razão da exceção.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-122">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="5d188-123">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" /> com uma mensagem de erro especificada.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-123">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class with a specified error message.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="5d188-124">Este construtor inicializa a propriedade <xref:System.Exception.Message%2A> da nova exceção usando-se o parâmetro `message`.</span><span class="sxs-lookup"><span data-stu-id="5d188-124">This constructor initializes the <xref:System.Exception.Message%2A> property of the new exception using the `message` parameter.</span></span> <span data-ttu-id="5d188-125">O conteúdo de `message` deve ser compreendido por humanos.</span><span class="sxs-lookup"><span data-stu-id="5d188-125">The content of `message` is intended to be understood by humans.</span></span> <span data-ttu-id="5d188-126">O chamador desse construtor é necessário para garantir que essa cadeia de caracteres tenha sido localizada para a cultura do sistema atual.</span><span class="sxs-lookup"><span data-stu-id="5d188-126">The caller of this constructor is required to ensure that this string has been localized for the current system culture.</span></span>  
  
 <span data-ttu-id="5d188-127">A tabela a seguir mostra os valores de propriedade inicial para a nova instância.</span><span class="sxs-lookup"><span data-stu-id="5d188-127">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="5d188-128">Propriedade</span><span class="sxs-lookup"><span data-stu-id="5d188-128">Property</span></span>|<span data-ttu-id="5d188-129">Valor</span><span class="sxs-lookup"><span data-stu-id="5d188-129">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="5d188-130">`null`.</span><span class="sxs-lookup"><span data-stu-id="5d188-130">`null`.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="5d188-131">A cadeia de caracteres da mensagem de erro.</span><span class="sxs-lookup"><span data-stu-id="5d188-131">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="protected AsynchronousOperationException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);" />
      <MemberSignature Language="ILAsm" Value=".method familyhidebysig specialname rtspecialname instance void .ctor(class System.Runtime.Serialization.SerializationInfo info, valuetype System.Runtime.Serialization.StreamingContext context) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.AsynchronousOperationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
      <MemberSignature Language="VB.NET" Value="Protected Sub New (info As SerializationInfo, context As StreamingContext)" />
      <MemberSignature Language="C++ CLI" Value="protected:&#xA; AsynchronousOperationException(System::Runtime::Serialization::SerializationInfo ^ info, System::Runtime::Serialization::StreamingContext context);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.AsynchronousOperationException : System.Runtime.Serialization.SerializationInfo * System.Runtime.Serialization.StreamingContext -&gt; System.IdentityModel.AsynchronousOperationException" Usage="new System.IdentityModel.AsynchronousOperationException (info, context)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="info" Type="System.Runtime.Serialization.SerializationInfo" />
        <Parameter Name="context" Type="System.Runtime.Serialization.StreamingContext" />
      </Parameters>
      <Docs>
        <param name="info">
          <span data-ttu-id="5d188-132">Um objeto <see cref="T:System.Runtime.Serialization.SerializationInfo" /> que contém os dados do objeto serializado.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-132">A <see cref="T:System.Runtime.Serialization.SerializationInfo" /> object that holds the serialized object data.</span>
          </span>
        </param>
        <param name="context">
          <span data-ttu-id="5d188-133">Um objeto <see cref="T:System.Runtime.Serialization.StreamingContext" /> que contém as informações contextuais sobre a origem ou o destino.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-133">A <see cref="T:System.Runtime.Serialization.StreamingContext" /> object that contains the contextual information about the source or destination.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="5d188-134">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" /> com dados serializados.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-134">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class with serialized data.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="5d188-135">Este construtor é chamado durante a desserialização para reconstituir o objeto de exceção transmitido por um fluxo.</span><span class="sxs-lookup"><span data-stu-id="5d188-135">This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.</span></span>  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AsynchronousOperationException (string message, Exception innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:System.IdentityModel.AsynchronousOperationException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="C++ CLI" Value="public:&#xA; AsynchronousOperationException(System::String ^ message, Exception ^ innerException);" />
      <MemberSignature Language="F#" Value="new System.IdentityModel.AsynchronousOperationException : string * Exception -&gt; System.IdentityModel.AsynchronousOperationException" Usage="new System.IdentityModel.AsynchronousOperationException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>System.IdentityModel</AssemblyName>
        <AssemblyVersion>4.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">
          <span data-ttu-id="5d188-136">A mensagem de erro que explica a razão da exceção.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-136">The error message that explains the reason for the exception.</span>
          </span>
        </param>
        <param name="innerException">
          <span data-ttu-id="5d188-137">A <see cref="T:System.Exception" /> que é a causa da exceção atual.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-137">The <see cref="T:System.Exception" /> that is the cause of the current exception.</span>
          </span>
          <span data-ttu-id="5d188-138">Caso o parâmetro <paramref name="innerException" /> não seja <see langword="null" />, a exceção atual é acionada em um bloco <see langword="catch" /> que identifica a exceção interna.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-138">If the <paramref name="innerException" /> parameter is not <see langword="null" />, the current exception is raised in a <see langword="catch" /> block that handles the inner exception.</span>
          </span>
        </param>
        <summary>
          <span data-ttu-id="5d188-139">Inicializa uma nova instância da classe <see cref="T:System.IdentityModel.AsynchronousOperationException" /> com uma mensagem de erro especificada e uma referência à exceção interna que é a causa da exceção.</span>
          <span class="sxs-lookup">
            <span data-stu-id="5d188-139">Initializes a new instance of the <see cref="T:System.IdentityModel.AsynchronousOperationException" /> class with a specified error message and a reference to the inner exception that is the cause of this exception.</span>
          </span>
        </summary>
        <remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 <span data-ttu-id="5d188-140">Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência para a exceção anterior na propriedade <xref:System.Exception.InnerException%2A>.</span><span class="sxs-lookup"><span data-stu-id="5d188-140">An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property.</span></span> <span data-ttu-id="5d188-141">O <xref:System.Exception.InnerException%2A> propriedade retorna o mesmo valor que é passado para o construtor.</span><span class="sxs-lookup"><span data-stu-id="5d188-141">The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor.</span></span>  
  
 <span data-ttu-id="5d188-142">A tabela a seguir mostra os valores de propriedade inicial para a nova instância.</span><span class="sxs-lookup"><span data-stu-id="5d188-142">The following table shows the initial property values for the new instance.</span></span>  
  
|<span data-ttu-id="5d188-143">Propriedade</span><span class="sxs-lookup"><span data-stu-id="5d188-143">Property</span></span>|<span data-ttu-id="5d188-144">Valor</span><span class="sxs-lookup"><span data-stu-id="5d188-144">Value</span></span>|  
|--------------|-----------|  
|<xref:System.Exception.InnerException%2A>|<span data-ttu-id="5d188-145">A referência de exceção interna.</span><span class="sxs-lookup"><span data-stu-id="5d188-145">The inner exception reference.</span></span>|  
|<xref:System.Exception.Message%2A>|<span data-ttu-id="5d188-146">A cadeia de caracteres da mensagem de erro.</span><span class="sxs-lookup"><span data-stu-id="5d188-146">The error message string.</span></span>|  
  
 ]]></format>
        </remarks>
      </Docs>
    </Member>
  </Members>
</Type>